CONTRIBUTING.md

```diff
@@ -47,6 +48,14 @@ Python 3.6.3 (default, Jul 17 2017, 16:44:45)
 >>> capture_snapshot_of_user_count()
 ```
 
+If you are using Docker Virtual Machine, you may run into an issue connecting to Postgres and Redis. To fix this you can override the default connection strings._
+
+```bash
+(.venv) $ docker-machine ls - This will display the IP address of the virtual machine
```

Comment:
Please replace `-` with `#` so it will show as a comment in bash syntax highlighting.